seatunnel-core/seatunnel-core-sql/pom.xml

```diff
@@ -66,5 +66,10 @@
             <artifactId>flink-streaming-scala_${scala.binary.version}</artifactId>
             <version>${flink.version}</version>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
```

Comment:
we can remove `<scope>test</scope>`

Reply:
ok

> Hi, can you format your code, there is a codestyle file in our tools directory, you can import it to check, or use `mvn checkstyle:check` to check

got it will change later

Reply:
done, please check